### PR TITLE
[Multi Cluster] format + fixes around the new shared section

### PR DESCRIPTION
**Pull request by ferantivero**
related: #374114

some formatting + fixes

### CODE REVIEW

File: 05-cluster-prerequisites.md [hunk 1]
```diff
@@ -35,7 +35,17 @@ Following the steps below will result in the provisioning of the shared Azure re
...
...ly emit logs to Azure Monitor. All the collected data is stored in a [centralized Log Analytics workspace for the ease of operations](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-overview) after confirming there is no need to split workspaces due to scale. The app team estimates that the ingestion rate is going to be less than `6GB/minute`, so they expect not to be throttled as this is supported by the default rate limit. If it was required, they could grow by changing this setup eventually. In other words, the design decision is to create a single Azure Log Analytics workspace instance in the `eastus2` region, and that is going to be shared among their multiple clusters. Additionally, there is no business requirement for a consolidated cross business units view at this moment, so the centralization is great option for them. Something the app team also considered while making a final decision is the fact that migrating from a _centralized_ solution to a _decentralized_ one can be much easier than doing it the other way around. As a result, the single workspace being created is a workload specific workspace, and these are some of the Azure services sending data to it:
```
Comment (ckittel) [inline 05-cluster-prerequisites.md:48]: > https://docs.microsoft.com/en-us/azure/frontdoor/front-door-overview

Wrong href on that Log Analytics link, yeah?

File: 05-cluster-prerequisites.md [hunk 2]
```diff
@@ -35,7 +35,17 @@ Following the steps below will result in the provisioning of the shared Azure re
...
+  > :book: The app team is about to provision three shared Azure resources. One is a non-regional and two regional, but more importantly they are deployed independently from their AKS Clusters:
+  >
+  > | Azure Resource                                                                                                | Non-Regional  |  East US 2  | Central US |
```
Comment (ckittel) [inline 05-cluster-prerequisites.md:40]: I'm totes okay with the regions being called out explicitly like this. It matches the contents, and honestly we don't really give users that much control over the regions.  I mean, they could change params, but we haven't really "tested" that functionality or provided guidance on exactly what to change where.  So I'm all for what ya have here.

**Review by ckittel — APPROVED** [review]
Just one small change -- bunk URL I think...

File: 05-cluster-prerequisites.md (hunk 2 above)
Comment (ferantivero) [reply 05-cluster-prerequisites.md:40]: thanks for this feedback. Please let me resolve this one based on this ^

File: 05-cluster-prerequisites.md (hunk 1 above)
Comment (ferantivero) [reply 05-cluster-prerequisites.md]: good catch, it is now addressed from [f9e29](https://github.com/mspnp/aks-secure-baseline/pull/172/commits/f9e29e4dbcde5e96c4488344db95491c0181fc59)

**Pull request merged**